src/consts/olm_operators.py

```diff
@@ -79,6 +80,11 @@ def values(cls) -> dict:
                 worker_count=4,
             ),
             OperatorType.LSO: cls.get_resource_dict(),
+            OperatorType.LVM: cls.get_resource_dict(
+                master_memory=1200,
```

Comment:
just so I understand, those are additional resources that are required for the operator right?